packages/amplify_authenticator/lib/src/widgets/form_fields/verify_user_form_field.dart

```diff
@@ -210,5 +197,5 @@ class _VerifyAttributeFieldState
   }
 
   @override
-  UsernameAttribute? selectionValue = _default;
```

Comment:
With the current implementation, `_default` is initialized to email, and then is updated on line 191. Line 191 is not running before selectionValue is set (unless the user goes logs out and goes through the flow a second time), causing selectionValue to always be email.